main.cpp

```diff
@@ -399,19 +417,53 @@ struct help_command : public cmd {
     }
 } help_cmd;
 
+struct version_command : public cmd {
+    version_command() : cmd("version") {}
+    bool execute(device_map &devices) override {
+        if (settings.version.semantic)
+            std::cout << PICOTOOL_VERSION << "\n";
+        else
+            std::cout << "picotool v" << PICOTOOL_VERSION << " (" << SYSTEM_VERSION << ", " << COMPILER_INFO << ")\n";
```

Comment:
Only a small nit-pick, but this displays e.g.
```
picotool v1.1.0-develop (Linux 5.4.0-81-generic, GNU-7.5.0 Release)
```
_Maybe_ there should be an additional comma between `GNU-7.5.0` and `Release` ?

Reply:
fixed